pages/contact/index.vue

```diff
@@ -0,0 +1,603 @@
+      })
+    }
+
+    getVisibleState(el) {
+      let { showProp, showValue, hideProp, hideValue } = el.dataset
+      const value = this.state.get(showProp || hideProp)
+      console.log({ showValue })
+
+      try {
+        showValue = JSON.parse(showValue)
+      } catch (e) {}
+      try {
+        hideValue = JSON.parse(hideValue)
+      } catch (e) {}
+      console.log({ showValue })
+
+      if (Array.isArray(showValue || hideValue)) {
+        return (showValue || hideValue).indexOf(value) > -1
+      }
+      return (showValue || hideValue) === value
+    }
+  }
+
+  export default {
+    methods: {
+      // Vue keeps mangling prop values that look like arrays, this method avoids that
+      arrayString(...args) {
+        return `["${args.join('", "')}"]`
+      }
+    },
+    mounted() {
+      const subTypeInput = this.$el.querySelector('#subType')
+
+      const sessionStorageAdapter = new SessionStorageAdapter({
+        storageKey: 'contact_form',
+      })
+
+      const formDataTracker = new FormDataTracker({
+        formSelector: '#coderdojo_contact',
+        storageAdapter: sessionStorageAdapter,
+      })
+
+      const visibilityManager = new VisibilityManager({
+        parentSelector: '#coderdojo_contact',
+        storageAdapter: sessionStorageAdapter,
+      })
+    }
+  }
+</script>
+
+
+<style lang="scss" scoped>
+  @import '~/assets/_variables.scss';
+
+  p, ul {
+    margin-bottom: 2rem;
+  }
+
+  .c-contact {
+    max-width: $content-width;
+    margin: 0 auto;
+
+    &__form {
+      max-width: 80rem;
+      margin: 2rem 0;
+    }
+  }
+
+  .c-form-fieldset__wrapper {
+    padding-inline: 1.6rem;
+    margin-bottom: .8rem;
+    -webkit-margin-after: .8rem;
+    margin-block-end: .8rem;
+    padding: 1.6rem;
+    padding-block: 1.6rem;
+    background-color: #f0f8ff;
+    border: 0;
+    display: flex;
+    flex-direction: column;
+  }
+
+  @media only screen and (min-width: 800px) {
+    .c-form-fieldset__wrapper {
+      padding-left: 7rem;
+      padding-right: 7rem;
+      padding-inline: 7rem;
+      margin-bottom: 1rem;
+      -webkit-margin-after: 1rem;
+      margin-block-end: 1rem;
+      padding-top: 3rem;
+      padding-bottom: 3rem;
+      padding-block: 3rem;
+    }
+  }
+
+  @media only screen and (min-width: 1200px) {
+    .c-form-fieldset__wrapper {
+      padding-left: 8.4rem;
+      padding-right: 8.4rem;
+      padding-inline: 8.4rem;
+      margin-bottom: 1.2rem;
+      -webkit-margin-after: 1.2rem;
+      margin-block-end: 1.2rem;
+      padding-top: 3.6rem;
+      padding-bottom: 3.6rem;
+      padding-block: 3.6rem;
+    }
+  }
+
+  .c-form-fieldset__element {
+    border: 0;
+    padding: 0;
+  }
+
+  .c-form-fieldset__legend {
+    font-size: 1.92rem;
+    -webkit-margin-after: 2.4rem;
+    margin-block-end: 2.4rem;
+    font-weight: 700;
+    margin-top: 0;
+    font-size: 2.4rem;
+    margin-bottom: .8rem;
+    -webkit-margin-after: .8rem;
+    margin-block-end: .8rem;
```

Comment:
Looks like some duplicate styles? Not sure how this has happened! Same for the `min-width: 800px` media query below.

Reply:
Good catch. Not sure what happened there exactly. Could've easily been human error.